lisa/tests/scheduler/eas_behaviour.py

```diff
@@ -469,8 +469,11 @@ class TwoBigTasks(EASBehaviour):
 
     @classmethod
     def get_rtapp_profile(cls, plat_info):
-        bigs = plat_info["capacity-classes"][-1]
-        duty = cls.unscaled_utilization(plat_info, bigs[0], 80)
+        num_classes = len(plat_info["capacity-classes"])
+        # A duty cycle equivalent to the max capacity for all CPUs != big
+        # is enough to ensure migration to bigs
+        pre_bigs = plat_info["capacity-classes"][-2 % num_classes]
+        duty = cls.unscaled_utilization(plat_info, pre_bigs[0], 100)
```

Comment:
We should add some margin on top. Having the mean util_avg equal to pre-big capa means the util_avg will jump around that value. We don't want the task to migrate back and forth as the value evolves (unless migration is triggered based on util_est_enqueued ?)

Reply:
That is the reason I've put a duty-cycle equivalent to 100% pre-big capacity. As long as the jumping around is in within of the 20% capacity margin in the scheduler
> #define fits_capacity(cap, max)»((cap) * 1280 < (max) * 1024)
I don't believe it will ping pong back and forth. 

I would rather not add an extra hard-coded margin here unless we find we need it.

Reply:
We can definitely get 10 to 20% of util variation because of CPU invariance not doing its job, plus the util swing that depends on the task period. We can expect ~7 or 8% (i.e. 80 or 100 PELT units) oscillation in util_avg as well. That means it's not very hard to have an overall 20% variation if the CPU capacities are not writeable, or if we end up increasing the period for better testing coverage.